json/pom.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2012-2021 The Feign Authors
+
+    Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+    in compliance with the License. You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software distributed under the License
+    is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+    or implied. See the License for the specific language governing permissions and limitations under
+    the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.github.openfeign</groupId>
+    <artifactId>parent</artifactId>
+    <version>-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>feign-json</artifactId>
```

Comment:
@kdavisk6 any opinion about the package/module names?  Just calling it "json" triggers something on the back of my mind, not that I have a better suggestion.

Reply:
Since the implementation here is `org.json`, I'm good with it.

Reply:
I have renamed it to `feign-org.json` and module to `org.json`.

Reply:
Ow, no, feign-json was fine

Reply:
Sorry, misunderstood

I has returned back feign-json as artifact name and json as directory name.

Reply:
How about json-java or org.json as folder/mofule name? The artifact name is still feign-json.

I also did not like that the name is too common, but the artifact of the library, like its name (json-java), is also too general and I could not choose another name.